app/assets/javascripts/modules/media_viewer.js

```diff
@@ -98,42 +157,23 @@
           var checkWindow = setInterval(function() {
             if (!_timedOut(start, 30000) &&
               (!windowReference || !windowReference.closed)) return;
-            clearInterval(checkWindow);
-            mediaObject
-              .parents(sliderObjectSelector)
-              .find('[data-auth-link], ' + restrictedOverlaySelector + ', ' + restrictedMessageSelector)
-              .hide();
-            // As best I can tell, .touch() does nothing (and does not exist in jQuery),
-            // but without calling some function after .load() the media fails to load.
-            mediaObject[0].load().touch();
+            authCheckForMediaObject(mediaObject, function(_, data) {
+              if(data.status === 'success') {
+                clearInterval(checkWindow);
+                mediaObject
+                  .parents(sliderObjectSelector)
+                  .find('[data-auth-link], ' + restrictedOverlaySelector + ', ' + restrictedMessageSelector)
+                  .hide();
```

Comment:
Ah, actually, now that I read that selector correctly, I see that it is doing both (I thought it was constructing a single selector, nut multiple).

This was required before since the act of clicking the link wouldn't trigger another call to the check the authentication status of the user (which now it does).  I'll see if I can change this so that we don't attempt to do that twice (although it is harmless).